Tests/Shared.Specs/DictionaryEquivalencySpecs.cs

```diff
@@ -786,10 +786,35 @@ public void When_two_dictionaries_asserted_to_be_equivalent_have_different_lengt
             // Assert
             //-----------------------------------------------------------------------------------------------------------
             act1.Should().Throw<XunitException>()
-                .WithMessage("Expected subject to be a dictionary with 2 item(s), but found 1 item(s)*");
+                .WithMessage("Expected subject to be a dictionary with 2 item(s), but found 1 item(s).*Missing key(s): {\"farewell\"}*");
             
             act2.Should().Throw<XunitException>()
-                .WithMessage("Expected subject to be a dictionary with 1 item(s), but found 2 item(s)*");
+                .WithMessage("Expected subject to be a dictionary with 1 item(s), but found 2 item(s).*Additional key(s): {\"farewell\"}*");
+        }
+
+        [Fact]
+        public void When_two_dictionaries_asserted_to_be_equivalent_have_different_lengths_with_both_missing_and_additional_items_it_should_fail_descriptively()
```

Comment:
This might be personal preference, but I would split this into two separate tests.

Reply:
Thanks, good idea, I did that.